__tests__/app.test.js

```diff
@@ -531,3 +531,60 @@ describe("DELETE /api/comments/:comment_id", () => {
     
   });
 });
+
+describe("GET /api/users", () => {
+  it("200: should respond with a 200 status code ", () => {
+    return request(app).get("/api/users").expect(200);
+  });
+
+  it("200: should return an array", () => {
+    return request(app)
+      .get("/api/users")
+      .then((data) => {
+        expect(Array.isArray(data.body)).toBe(true);
+      });
+  });
+
+  it("200 should respond with an array of all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const users = body;
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url : expect.any(String)
+          });
+        });
+      });
+  });
+
+  // it("200 should respond with an empty array if the users folder is empty", () => {
+  //   return connection
+  //     .query("DELETE FROM comments")
+  //     .then(() => {
+  //       return connection.query("DELETE FROM articles");
+  //     })
+  //     .then(() => {
+  //       return connection.query("DELETE FROM users");
+  //     })
+  //     .then(() => {
+  //       return request(app).get("/api/users").expect(200);
+  //     })
+  //     .then((data) => {
+  //       expect(data.body).toEqual([]);
+  //     });
+  // });
```

Comment:
Generally, when something is commented out and you don't want to use it, you should be deleting it before committing and pushing. When you are asking for a PR to be done on your code, you are saying "this piece of work is done, it is ready to be reviewed" - big commented out blocks of code shouldn't stay in the code